chef-config/lib/chef-config/dist.rb

```diff
@@ -0,0 +1,8 @@
+module ChefConfig
+  class Dist
+    # The chef executable name. Also used in directory names.
+    EXEC = "chef".freeze
```

Comment:
We shouldn't need this here. The path below isn't based on the executable name. It's really based on the company name. I'd just let CONF_DIR equal a string and avoid basing it on anything else.